scripts/installer/glassfish-setup.sh

```diff
@@ -157,121 +157,128 @@ if [  $(echo $DOMAIN_DOWN|wc -c) -ne 1  ];
+  # undeploy the app, if running: 
 
-# avoid OutOfMemoryError: PermGen per http://eugenedvorkin.com/java-lang-outofmemoryerror-permgen-space-error-during-deployment-to-glassfish/
-#./asadmin $ASADMIN_OPTS list-jvm-options
-./asadmin $ASADMIN_OPTS delete-jvm-options "-XX\:MaxPermSize=192m"
-./asadmin $ASADMIN_OPTS create-jvm-options "-XX\:MaxPermSize=512m"
-./asadmin $ASADMIN_OPTS create-jvm-options "-XX\:PermSize=256m"
-./asadmin $ASADMIN_OPTS delete-jvm-options -Xmx512m
-./asadmin $ASADMIN_OPTS create-jvm-options "-Xmx${MEM_HEAP_SIZE}m"
-./asadmin $ASADMIN_OPTS delete-jvm-options -client
-./asadmin $ASADMIN_OPTS create-jvm-options "-server"
+  ./asadmin $ASADMIN_OPTS undeploy dataverse-4.0
 
-###
-# JDBC connection pool
+  # avoid OutOfMemoryError: PermGen per http://eugenedvorkin.com/java-lang-outofmemoryerror-permgen-space-error-during-deployment-to-glassfish/
+  #./asadmin $ASADMIN_OPTS list-jvm-options
+  ./asadmin $ASADMIN_OPTS delete-jvm-options "-XX\:MaxPermSize=192m"
+  ./asadmin $ASADMIN_OPTS create-jvm-options "-XX\:MaxPermSize=512m"
+  ./asadmin $ASADMIN_OPTS create-jvm-options "-XX\:PermSize=256m"
+  ./asadmin $ASADMIN_OPTS delete-jvm-options -Xmx512m
+  ./asadmin $ASADMIN_OPTS create-jvm-options "-Xmx${MEM_HEAP_SIZE}m"
+  ./asadmin $ASADMIN_OPTS delete-jvm-options -client
+  ./asadmin $ASADMIN_OPTS create-jvm-options "-server"
 
-# we'll try to delete a pool with this name, if already exists. 
-# - in case the database name has changed since the last time it 
-# was configured. 
-./asadmin $ASADMIN_OPTS delete-jdbc-connection-pool --cascade=true dvnDbPool
+  ###
+  # JDBC connection pool
 
+  # we'll try to delete a pool with this name, if already exists. 
+  # - in case the database name has changed since the last time it 
+  # was configured. 
+  ./asadmin $ASADMIN_OPTS delete-jdbc-connection-pool --cascade=true dvnDbPool
 
-./asadmin $ASADMIN_OPTS create-jdbc-connection-pool --restype javax.sql.DataSource \
-                                      --datasourceclassname org.postgresql.ds.PGPoolingDataSource \
-                                      --property create=true:User=$DB_USER:PortNumber=$DB_PORT:databaseName=$DB_NAME:password=$DB_PASS:ServerName=$DB_HOST \
-                                      dvnDbPool
 
-###
-# Create data sources
-./asadmin $ASADMIN_OPTS create-jdbc-resource --connectionpoolid dvnDbPool jdbc/VDCNetDS
+  ./asadmin $ASADMIN_OPTS create-jdbc-connection-pool --restype javax.sql.DataSource \
+                                        --datasourceclassname org.postgresql.ds.PGPoolingDataSource \
+                                        --property create=true:User=$DB_USER:PortNumber=$DB_PORT:databaseName=$DB_NAME:password=$DB_PASS:ServerName=$DB_HOST \
+                                        dvnDbPool
 
-###
-# Set up the data source for the timers
+  ###
+  # Create data sources
+  ./asadmin $ASADMIN_OPTS create-jdbc-resource --connectionpoolid dvnDbPool jdbc/VDCNetDS
```

Comment:
Are the changes above just whitespace changes?

Reply:
No, in this scripts, JVM setup is different for container because works were done in Dockerfile